examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL/SDL.h>
```

Comment:
Give me a sec I'll test it on something that doesnt have it, not even SDL1.2 (which is the only system wide SDL that I have)